packages/uniforms/src/types.ts

```diff
@@ -30,6 +30,11 @@ export type DeepPartial<T> = {
   [P in keyof T]?: T[P] extends {} ? DeepPartial<T[P]> : T[P];
 };
 
+export type FieldProps<Value, Base, Extension = {}> = Override<
+  Base,
+  GuaranteedProps<Value> & Extension
+>;
+
```

Comment:
Why `FieldProps` If its only used in HTMLFieldProps?

Reply:
It's going to be used in `uniforms-antd` and `uniforms-material` as they use custom components.